inductiva/simulators/dummy_simulator.py

```diff
@@ -0,0 +1,36 @@
+"""Dummy simulator for demonstrating and testing purposes.
+
+This simulator acts as an echo of the arguments passed and can be used to
+test the commands passed to the simulator.
+"""
+
+from typing import Optional
+
+from inductiva import types, tasks, simulators
+
+
+class DummySimulator(simulators.Simulator):
+    """Dummy simulator for demo and testing purposes."""
+
+    def __init__(self):
+        super().__init__()
+        self.api_method_name = "tester.echo.run_simulation"
+
+    def run(self,
+            input_dir: types.Path,
+            input_filename: str,
+            sleep_time: Optional[float] = 1,
+            extra_metadata: Optional[dict] = None) -> tasks.Task:
+        """Run a dummy simulation.
```

Comment:
Maybe, instead of "Run a dummy simulation", specify that it just `echo`s stuff to file.